docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -35,112 +32,122 @@ documented on the identity providers website. Here are a few links:
+example commands below:
+
+- Client ID: <Var name="<CLIENT-ID>" description="The Client ID for your Teleport application, provided by your IdP"/>
+- Client secret: <Var name="<CLIENT-SECRET>" description="The Client secret for your Teleport application, provided by your IdP"/>
+
 ## OIDC Redirect URL
 
-OIDC relies on HTTP re-directs to return control back to Teleport after
+OIDC relies on HTTP redirects to return control back to Teleport after
 authentication is complete. The redirect URL must be selected by a Teleport
 administrator in advance.
 
-If the Teleport web proxy is running on `proxy.example.com` host, the redirect URL
-should be `https://proxy.example.com:3080/v1/webapi/oidc/callback`
+The redirect URL for OIDC authentication in Teleport is <nobr><Var
+name="mytenant.teleport.sh" description="Your Teleport Cloud tenant or Proxy
+Service address"/>`/v1/webapi/oidc/callback`</nobr>. Replace `mytenant.teleport.sh`
+with your Teleport Cloud tenant or Proxy Service address.
 
 ## OIDC connector configuration
 
-The next step is to add an OIDC connector to Teleport. The connectors are manipulated
-via `tctl` [resource commands](../../reference/resources.mdx). To create a new connector,
-create a connector resource file in YAML format, for example `oidc-connector.yaml`.
+The next step is to add an OIDC connector to Teleport. The connectors are
+created, test, and added / removed via `tctl` [resource commands](../../reference/resources.mdx).
 
-The file contents are shown below. This connector requests the scope `group`
-from the identity provider then mapping the value to either to `admin` role or
-the `user` role depending on the value returned for `group` within the claims.
-
-```yaml
-(!examples/resources/oidc-connector.yaml!)
-```
-
-Create the connector:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+To create a new connector, use `tctl sso configure`. Our example creates a
+connector resource file in YAML format named `oidc-connector.yaml`:
 
 ```code
-# Log in to your cluster with tsh so you can use tctl from your local machine.
-# You can also run tctl on your Auth Service host without running "tsh login"
-# first.
-$ tsh login --proxy=teleport.example.com --user=myuser
-$ tctl create oidc-connector.yaml
+$ tctl sso configure oidc --name <CONNECTOR-NAME> \
+  --issuer-url <PATH-TO-PROVIDER> \
+  --id <CLIENT-ID> --secret <CLIENT-SECRET> \
+  --claims-to-roles <CLAIM-KEY>,<CLAIM-VALUE>,access \
+  --claims-to-roles <CLAIM-KEY>,<CLAIM-VALUE>,editor > oidc-connector.yaml
 ```
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
+- `--name`: Usually the name of the IdP, this is how the connector will be
+  identified in Teleport.
+- `--issuer-url`: This is the base path to the IdP's OIDC configuration endpoint,
+  excluding `.well-known/openid-configuration`. If, for example, the endpoint
+  is `https://example.com/.well-known/openid-configuration`, you would use
+  `example.com`.
```

Comment:
Wouldn't it be https://example.com then? (example.com is not a URL, nor does it match the keycloak example below)

Reply:
```suggestion
  `https://example.com`.
```